src/tools/language-server/tsconfig.json

```diff
@@ -11,6 +11,7 @@
   },
   "exclude": [
     "../../../build/**/*",
-    "../../../server/**/*"
+    "../../../server/**/*",
+    "../**/*"
```

Comment:
Does this exclude everything, is that what we want to do?

Reply:
Thanks, I had thought this would exclude everything expect for the manually included code and then had a false positive when testing it. I've changed it to exclude the vscode client (which was the actual code I needed to not be included).